gcm_filters/kernels.py

```diff
@@ -91,6 +93,108 @@ def __call__(self, field: ArrayType):
 ALL_KERNELS[GridType.CARTESIAN_WITH_LAND] = CartesianLaplacianWithLandMask
 
 
+@dataclass
+class POPTripolarSimpleLaplacian(CartesianLaplacianWithLandMask):
+    """̵Laplacian for POP tripolar grid geometry with land mask, but assuming that dx = dy = 1
+
+    Attributes
+    ----------
+    wet_mask: Mask array, 1 for ocean, 0 for land
+    position: string
+        - 'T': Laplacian is applied for field defined at T-points
+        - 'U': Laplacian is applied for field defined at U-points
```

Comment:
Rather than having a "position" here, I imagined that these would actually be completely separate kernels, i.e.
- `POPTripolarSimpleLaplacianTpoint`
- `POPTripolarSimpleLaplacianUpoint`

Reply:
Thanks @rabernat. This was exactly a question I had for you. Having separate kernels will make things a lot easier in other parts of the code.

Reply:
I will try to finish up this PR this afternoon, and let you know when it's ready to review.